docs/faking-async-methods.md

```diff
@@ -27,3 +27,19 @@ Of course, you can still configure calls to `async` methods as you would normall
 ```csharp
 A.CallTo(() => foo.Bar()).Returns(Task.FromResult("bar"));
 ```
+
+There are also convenience overloads of `Returns` and `ReturnsLazily` that let you specify a value rather
+than a task, and configure the method to return a completed task whose result is the specified value:
+
+```csharp
+A.CallTo(() => foo.Bar()).Returns("bar");
+```
+
+These overloads of `Returns` and `ReturnsLazily` also exist for `ValueTask<T>`; however, unless you're
+using the .NET Standard 2.1 version of FakeItEasy, they're in a separate package:
+[`FakeItEasy.Extensions.ValueTask`](https://www.nuget.org/packages/FakeItEasy.Extensions.ValueTask).

```

Comment:
This is a good note. However, I worry that we'll forget to update it when we add a .NET Standard 3.2 package or whatever. Would saying that you need to use the new package whenever using a version of FakeItEasy that targets a framework that lacks an inbuilt `ValueTask` make sense?

Reply:
Good point. I'll do that.

Reply:
In fact, it's not so simple... .NET Core 1.0+ has `ValueTask<T>`, and .NET Core 2.1+ has `ValueTask`. But if the user's project targets .NET Core 2.1, it will pick the .NET Standard 2.0 version of FakeItEasy, which doesn't have the methods (since they're only in the .NET Standard 2.1 version). I'll try to think of an explanation that makes sense.

Reply:
Done. It's not ideal, but I can't think of a better way to phrase it that is both accurate and easy to understand.

Reply:
Lovely. Thanks.